releasenotes/notes/agent_remove_max_wokers_hard_limit-41fd2c1873d89549.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Remove agent max workers hard limit
```

Comment:
We have many types of workers in the agent, I think we should be more precise about which ones we're talking about.

Also could we explain in which case a users might want to increase this past 25.

Reply:
Sure improved the release note here: https://github.com/DataDog/datadog-agent/pull/7507/commits/7f4501824c64346600b5f91ecfacf8ab3f7b64a7